mlflow/tracking/_tracking_service/client.py

```diff
@@ -80,7 +81,38 @@ def get_metric_history(self, run_id, key):
 
         :return: A list of :py:class:`mlflow.entities.Metric` entities if logged, else empty list
         """
-        return self.store.get_metric_history(run_id=run_id, metric_key=key)
+
+        # NB: Paginated query support is currently only available for the RestStore backend.
+        # FileStore and SQLAlchemy store do not provide support for paginated queries and will
+        # raise an MlflowException if the `page_token` argument is not None when calling this
+        # API for a continuation query.
+
+        max_results = GET_METRIC_HISTORY_MAX_RESULTS
```

Comment:
```suggestion
```

Do we need this variable? Can we pass `GET_METRIC_HISTORY_MAX_RESULTS` to `get_metric_history` directly?

Reply:
```python
# We need to look around this line to find where max_resulsts comes from
self.store.get_metric_history(max_results=max_results, ...)

# It's clear that we call `self.store.get_metric_history` with a constant
self.store.get_metric_history(max_results=GET_METRIC_HISTORY_MAX_RESULTS, ...)
```

Reply:
changed!